packages/react-core/src/components/DualListSelector/DualListSelector.tsx

```diff
@@ -271,16 +275,20 @@ export class DualListSelector extends React.Component<DualListSelectorProps, Dua
     this.setState(prevState => {
       const movedOptions =
         prevState.availableTreeFilteredOptions ||
-        flattenTreeWithFolders(prevState.availableOptions as DualListSelectorTreeItemData[]);
+        flattenTreeWithFolders((prevState.availableOptions as unknown) as DualListSelectorTreeItemData[]);
```

Comment:
Curious.  Why `unknown` here and not `any`?

Reply:
The `unknown` keyword still allows a stricter subset of type checking, whereas `any` disables all checks. But in this case I used `unknown` as it was the recommendation by the compiler.